src/Sylius/Behat/Context/Setup/ChannelContext.php

```diff
@@ -127,6 +128,26 @@ public function theChannelIsDisabled(ChannelInterface $channel)
     }
 
     /**
+     * @Given /^(this channel) has (currency "([^"]+)")$/
+     */
+    public function thisChannelHasCurrency(ChannelInterface $channel, CurrencyInterface $currency)
+    {
+        $channel->addCurrency($currency);
+        $this->channelManager->flush();
+        $this->sharedStorage->set('channel', $channel);
+    }
+
+    /**
+     * @Given /^(this channel) host is "([^"]+)"$/
+     */
+    public function thisChannelHostIs(ChannelInterface $channel, $hostName)
```

Comment:
Yup, but before I couldn't  open my taxon show page with configuration like this from channel feature. Now is ok and I can remove this steps.